schema.graphql

```diff
@@ -253,12 +253,11 @@ type FpmmTransaction @entity {
   user: Account!
   fpmmType: FpmmType!
   transactionType: TransactionType!
-  collateralAmount: BigInt!
-  collateralAmountUSD: BigDecimal!
-  collateralTokenAddress: Bytes!
   collateralTokenAmount: BigInt!
+  sharesOrPoolTokenAmount: BigInt!
   creationTimestamp: BigInt!
   transactionHash: Bytes!
+  blockNumber:BigInt!
```

Comment:
Just out of curiosity, how does including the block number solve the problem of tracking those behind the scenes buys/sells?

Reply:
So i need to see at what block the transaction was executed and with that blockNumber i query past blockchain state for pool to determine the buy/sell at the time and this saves me one extra step of getting blockNumber from transaction hash instead...